components/modules/net.c

```diff
@@ -140,15 +129,18 @@ lnet_userdata *net_create( lua_State *L, enum net_type type ) {
 
   ud->type = type;
   ud->self_ref = LUA_NOREF;
-  ud->pcb = NULL;
+  ud->netconn = NULL;
 
   switch (type) {
     case TYPE_TCP_CLIENT:
       ud->client.cb_connect_ref = LUA_NOREF;
       ud->client.cb_reconnect_ref = LUA_NOREF;
       ud->client.cb_disconnect_ref = LUA_NOREF;
       ud->client.hold = 0;
+      ud->client.num_held = 0;
+      ud->client.connecting = false;
```

Comment:
Should `ud->client.closing` also be initialised here, or is that only relevant for UDP?